src/avro/src/schema.rs

```diff
@@ -2341,6 +2371,80 @@ mod tests {
         assert!(reader.is_empty()); // all bytes should have been consumed
     }
 
+    #[test]
+    fn default_non_nums() {
```

Comment:
Did you want to add a second test for `float` in addition to `double`?

Reply:
I would, but I don't want to go through another CI pass since this is actively blocking a user